src/tokenized-line.coffee

```diff
@@ -118,20 +118,27 @@ class TokenizedLine
     oddIndentLevel = @indentLevel - Math.floor(@indentLevel)
     Math.round(@tabLength * oddIndentLevel)
 
-  buildSoftWrapIndentationTokens: (token) ->
+  buildSoftWrapIndentationTokens: (token, hangingIndentationSpaces) ->
```

Comment:
What's your reasoning behind breaking the "hanging" indentation into a separate token rather than just making the initial indentation token longer?